03week/ticTacToe.js

```diff
@@ -23,26 +23,66 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+// Check for winning horizontal with if
+// 00 01 02
+// 10 11 12
+// 20 21 22
+const horizontalWin = () => {
+  if ((board[0][0] === board[0][1] && board[0][1] === board[0][2]) || (board[1][0] === board[1][1] && board[1][1] === board[1][2]) || (board[2][0] === board[2][1] &&
```

Comment:
Yes! Yes yes! You got the short way to reference three cells. Nice work! I mean, SOLID work!